### ISSUE TITLE
[BUG] Buffs in combat dissapear control

### ISSUE THREAD

**Issue by MichalNemec** [GoldpawsStuff/AzeriteUI5#26]
### Game Flavor

Wrath Classic

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Whenever you enter combat, your buffs dissapear. There is no way to have it always enabled.

### To reproduce

get a buff (tested with power word:shield) and get into combat, it dissapears, after combat leave, it reappears.

### Expected behavior

have control over visibility.

### Additional context

Its present on wow classic (non wotlk) too.

in my wtf/account/number/savedvariables/azeriteui.lua i have, because i tried /show, based on sourcecode.
```
["Auras"] = {
			["profiles"] = {
				["Azerite"] = {
					["alwaysShowAuras"] = true,
				},
			},
		},
```

**Comment by Goldpaw** [comment]
There are more options coming in 5.1, but it is currently working as intended, this is not a bug. 

The options you are showing from the wtf folder's saved settings, is not related to the auras found directly on the player frame, it's the auras found in the upper right corner of the screen. 

The upper right corner auras will always be hidden if you have a target selected, as they take up the same space on the screen. There are more options for this coming in 5.1, though. 

The auras found above the player's large unitframe, they are intentionally filtered in combat. Meaning any aura with a very long total duration and a lot of time remaining will be hidden during combat. The limit for duration or time left is currently set to 5 minutes, so anything with less than 5 minutes left should show up. 

No idea why this happens to power word: shield, though. In wrath this is a 30 second buff, so it should always be visible.

**Comment by Goldpaw** [comment]
There are aura visibility options in the new options menu available with the command `/azerite` in version 5.1, which is out now on the regular download locations.
